app/layout.tsx

```diff
@@ -23,7 +23,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en">
+    <html lang="en" className="overflow-hidden">
```

Comment:
Why do this `overflow-hidden`?

Reply:
`overflow-hidden` added here disables the scrolling on the site, this is useful because I don't want the left side of the screen to scroll vertically and I don't need the page to overflow and scroll horizontally either. I still want the right side of the site to be able to scroll vertically to view projects. `overflow-y-scroll` has been added to the `<motion.div>` that handles the right side of the page content. This way you can scroll through the projects without scrolling through the content on the left of the screen.

Reply:
After discussing this with you, I have removed this opting instead for using `fixed` to position content on the left side of the page to keep it from scrolling.
Thank you!

Reply:
Thanks for closing the loop on this.